ajax/libs/mediaelement-plugins/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "mediaelement-plugins",
+  "version": "1.0.0",
```

Comment:
please remove the `version`.
we mention it in https://github.com/cdnjs/cdnjs#adding-a-library-by-a-single-packagejson